lib/nodejs/authentication.js

```diff
@@ -133,7 +134,7 @@ router.post( "/logout", function( req, res ) {
 // for `/login` and `/logout`.
 
 router.use( function( req, res, next ) {
-  if ( req.isUnauthenticated() ) {
+  if ( req.isUnauthenticated() && !vwf.IsExempt( req ) ) {
```

Comment:
This works and I don't see a better option, but I will take the opportunity to point out that this is tacking parts of the old server to the new server and points toward worse maintainability. Hopefully the Unity client can log in through normal channels at some point and this connection can be removed.

The newer Express server handles some top-level requests and allows the rest to pass to the older, hand-coded server. Over time we'd like to move parts of the old server into Express until the old server eventually evaporates. Ideally the new server would never know about the old one--communicating only by declining to handle the routes it doesn't know about yet.

In that scheme, the SandTable routes could avoid the authorization check by moving them in front of the check without requiring authorization to be aware of the application. Unfortunately, that's impractical without moving most of the app to Express, which is a big leap. So, :+1:.

Reply:
Understood.  Thanks for your help in brainstorming solutions last week too!  I'll add your comment to the description of the story in JIRA (https://lmco-jiradc.us.lmco.com:8443/browse/ADSUDM-130)